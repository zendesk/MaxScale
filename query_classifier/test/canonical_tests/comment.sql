select 1;-- comment after statement
select 1;# comment after statement
select /* inline comment */ 1;
select /*! 1 + */ 1;
select /*!300000 1 + */ 1;
select /*!300000 1 + */ 1;
SELECT 2 /* +1 */;
SELECT 1 /*! +1 */;
SELECT 1 /*!50101 +1 */;
SELECT 2 /*M! +1 */;
SELECT 2 /*M!50101 +1 */;
