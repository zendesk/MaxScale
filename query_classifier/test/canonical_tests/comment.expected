select ?;
select ?;
select  ?;
select /*! ? + */ ?;
select /*!? ? + */ ?;
select /*!? ? + */ ?;
SELECT ? ;
SELECT ? /*! +? */;
SELECT ? /*!? +? */;
SELECT ? /*M! +? */;
SELECT ? /*M!? +? */;
